modules/oe_translation_remote/tests/modules/oe_translation_remote_test/src/EventSubscriber/TranslationSourceEventSubscriber.php

```diff
@@ -37,7 +37,7 @@ public function save(TranslationSourceEvent $event): void {
 
     // Get the #fake_value from the original data and set it onto a node title
     // so we can assert it in the test.
-    if ($node instanceof NodeInterface) {
+    if ($node instanceof NodeInterface && (bool) \Drupal::state()->get('oe_translation_remote_test_set_fake_value_on_node')) {
```

Comment:
Seems this would keep altering titles in tests needlessly and break certain assertions. So I locked it to only work on the specific test.